extension/sqlsmith/statement_generator.cpp

```diff
@@ -73,13 +88,29 @@ std::shared_ptr<GeneratorContext> StatementGenerator::GetDatabaseState(ClientCon
 			result->tables_and_views.push_back(entry);
 		});
 	}
+	if (context.transaction.HasActiveTransaction()) {
```

Comment:
Can you test to see what happens if you take this out? I remember adding it, but I don't think it's needed anymore. I added it for the attach statement

Reply:
Looks like it doest the same with and without these lines

Reply:
It is strange, but without this check test produces sometimes the ```EXECUTION ERROR: INTERNAL Error: DebugClientContextState::QueryEnd called when no query is active```

Reply:
ok, then you can leave it in